### PR TITLE
Major plotting tweaks

### PR DESCRIPTION
**Pull request by efriman**
-Added lineplot=True or --lineplot option when plotting stripes to add average lineplot over heatmap. Only works for one plot at a time
-Added colnames and rownames arguments to manually rename
-Stripes no longer load when not plotting stripes from CLI, more efficient
-Fixed bug with cols/rows that were not strings in CLI
-Fixed bug when plotting pileups of different flanks and/or resolutions
-Logs a warning when using sym=True and scale="linear", which are incompatible

### CODE REVIEW

File: coolpuppy/coolpup.py [hunk 1]
```diff
@@ -130,19 +130,20 @@ def load_pileup_df(filename, quaich=False):
...
+                    vstripe = "vertical_stripe_" + str(i)
+                    hstripe = "horizontal_stripe_" + str(i)
+                    coords = "coordinates_" + str(i)
+                    vertical_stripe.append(f[vstripe][:].toarray())
+                    horizontal_stripe.append(f[hstripe][:].toarray())
+                    coordinates.append(f[coords][:].astype("U13"))
+                annotation["vertical_stripe"] = vertical_stripe
+                annotation["horizontal_stripe"] = horizontal_stripe
+                annotation["coordinates"] = coordinates
+            except:
```
Comment (Phlya) [inline coolpuppy/coolpup.py:145]: Maybe add a specific exception that you want to ignore?

File: plotpuppy/CLI.py [hunk 2]
```diff
@@ -83,6 +83,12 @@ def parse_args_plotpuppy():
         required=False,
         help="""Whether to sort stripe stackups by total signal (sum), central pixel signal (center_pixel), or not at all (None)""",
     )
+    parser.add_argument(
+        "--lineplot",
+        default=False,
+        action="store_true",
+        help="""Whether to plot the average lineplot above stripes""",
+    )
```
Comment (Phlya) [inline plotpuppy/CLI.py:91]: Mention that it only works with a single plot?

File: plotpuppy/CLI.py [hunk 3]
```diff
@@ -281,6 +304,21 @@ def _excepthook(exc_type, value, tb):
     else:
         score = False
 
+    if args.cols:
+        if args.col_order:
+            col_order = args.col_order
```
Comment (Phlya) [inline plotpuppy/CLI.py:309]: Why do you need a new variable for this?

File: coolpuppy/coolpup.py (hunk 1 above)
Comment (efriman) [reply coolpuppy/coolpup.py:145]: How do you mean?

Comment (Phlya) [reply coolpuppy/coolpup.py:145]: except KeyError, or smth like this... To make sure some other unexpected error is not ignored

File: plotpuppy/CLI.py (hunk 3 above)
Comment (efriman) [reply plotpuppy/CLI.py:309]: That's a mistake. Fixed

**Pull request merged**